Remotion/Data/DomainObjects.UnitTests/StandardMappingTest.cs

```diff
@@ -181,9 +181,14 @@ protected string GetPropertyIdentifier (Type declaringType, string shortProperty
       return declaringType.FullName + "." + shortPropertyName;
     }
 
-    protected ClassDefinition GetTypeDefinition (Type classType)
+    protected ClassDefinition GetClassDefinition (Type classType)
```

Comment:
We need to introduce a wrapper for ClassDefinitionObjectMother that uses TypeDefinition as the public API. The goal is to only use GetClassDefinition or ClassDefinitionObjectMother where we actually want ClassDefinition instead of TypeDefinition.